documentation/multiple-forms-of-artifact.md

```diff
@@ -0,0 +1,113 @@
+# Different forms of generated artifact
+
+The Artifact Generator can generate many forms of artifacts, ranging from
+simple that only provide vocabulary term identifiers as string literals,
+through forms that provide those identifiers as IRI types from commonly used
+RDF libraries, all the way to forms that provide full access to the meta-data
+associated with individual vocabulary terms that may have been helpfully
+provided by the vocabulary creator.
+
+In the following examples, we show the generator producing a JavaScript
+constants to represent the `Person` class from Schema.org (i.e.,
+`https://schema.org/Person`).
```

Comment:
The following examples actually refer to FOAF rather than schema.org, is that intentional?

Reply:
Nope, not intentional - thanks! 59061b2